# Target repository: frameworks/base
# Make only com.google.android.gms queryable through PackageManager.

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -2004,5 +2004,6 @@
     <string-array name="config_forceQueryablePackages" translatable="false">
         <item>com.android.settings</item>
         <item>com.android.providers.settings</item>
+        <item>com.google.android.gms</item>
         <!-- Add packages here -->
     </string-array>
